XML/Other/ReplaceableTestParameters.xml

```diff
@@ -366,7 +366,7 @@ Scenario 2 : You need to run all the performance tests quickly.
 	</Parameter>
 	<Parameter>
 		<ReplaceThis>NTTTCP_VERSION</ReplaceThis>
-		<ReplaceWith>v1.3.4</ReplaceWith>
+		<ReplaceWith>master</ReplaceWith>
```

Comment:
@lizzha @LiliDeng  I just realized this from the talk in today's call.
The decision to go with a specific version was that we know that remains constant across runs.

a newer ntttcp version might give different throughput numbers, so we won't know that that is the key factor that can influence the numbers.
sticking to a specific version for a longer period of time would be more helpful for stability purposes of the results.

this is just on how I recall this change being done.

Reply:
understood, @LiliDeng simon will release ntttcp 1.3.5 version, could you update it to version 1.3.5?

Reply:
> understood, @LiliDeng simon will release ntttcp 1.3.5 version, could you update it to version 1.3.5?

Got it, will keep an eye on [Here](https://github.com/microsoft/ntttcp-for-linux/releases)